ios/brave-ios/Sources/Brave/Frontend/Brave Translate/BraveTranslateTabHelper.swift

```diff
@@ -325,6 +325,12 @@ class BraveTranslateTabHelper: NSObject {
       state: isTranslationSupported ? .available : .unavailable
     )
 
+    // Translation is not supported on this page, so do not show onboarding
+    // Return immediately
+    if !isTranslationSupported {
```

Comment:
This makes sure we don't translate from `en` to `en` for example. So a page cannot be translated if it's already in your desired language.